### PR TITLE
feat(rest-api-client): update space.getSpace method

### PR DESCRIPTION
**Pull request by shabaraba**
## Why

With the official release of the June 2024 version, the 'permissions' parameter was added in the response body of Get Space API.

## What

- Update the type of Space to add 'permissions' parameter.
  - In addition, the following parameters were missing, so they were also added to the type definition:
    - showAnnouncement
    - showThreadList
    - showAppList
    - showMemberList
    - showRelatedLinkList
- update doc for space.getSpace.

## How to test

1. Create a space on UI.
2. Exec space.getSpace method
3. Check to include the `permissions` parameter in the response
4. Check to be able to access `permissions` parameter of the value of the response.
5. 
## Checklist

- [x] Read [CONTRIBUTING.md](https://github.com/kintone/js-sdk/blob/main/CONTRIBUTING.md)
- [x] Updated documentation if it is required.
- [x] Added tests if it is required.
- [x] Passed `pnpm lint` and `pnpm test` on the root directory.

### CODE REVIEW

File: packages/rest-api-client/src/client/types/space/index.ts [hunk 1]
```diff
@@ -31,6 +35,12 @@ export type Space = {
   isGuest: boolean;
   attachedApps: AttachedApp[];
   fixedMember: boolean;
+  showAnnouncement: boolean;
+  showThreadList: boolean;
+  showAppList: boolean;
+  showMemberList: boolean;
+  showRelatedLinkList: boolean;
+  permissions: Permissions;
```
Comment (shabaraba) [inline packages/rest-api-client/src/client/types/space/index.ts:43]: `permissions`パラメータに対して`Permission`型だと配列で返りそうに受け取られるので、型名も複数形にしています。

Comment (tasshi-me) [reply packages/rest-api-client/src/client/types/space/index.ts:43]: 良さそうです！

File: packages/rest-api-client/src/client/types/space/index.ts [hunk 2]
```diff
@@ -31,6 +35,12 @@ export type Space = {
   isGuest: boolean;
   attachedApps: AttachedApp[];
   fixedMember: boolean;
+  showAnnouncement: boolean;
+  showThreadList: boolean;
+  showAppList: boolean;
+  showMemberList: boolean;
+  showRelatedLinkList: boolean;
```
Comment (tasshi-me) [inline packages/rest-api-client/src/client/types/space/index.ts:42]: これらのプロパティはnullableなので`boolean | null`が良さそうです！
https://kintone.dev/en/docs/kintone/rest-api/spaces/get-space/#response-parameters

**Review by tasshi-me — CHANGES_REQUESTED** [review]
1箇所だけ変更依頼を追加しました！

File: packages/rest-api-client/src/client/types/space/index.ts (hunk 2 above)
Comment (shabaraba) [reply packages/rest-api-client/src/client/types/space/index.ts]: [6b6bf97](https://github.com/kintone/js-sdk/pull/2876/commits/6b6bf977a5119f8255904562e0ea527beb61c99f)

**Comment by shabaraba** [comment]
@tasshi-me  
ありがとうございます、見逃していました :bow:

**Review by tasshi-me — APPROVED** [review]
LGTM!

**Pull request merged**